autoscraper/auto_scraper.py

```diff
@@ -107,16 +104,17 @@ def build(self, url=None, wanted_list=None, html=None, request_args=None):
 
         return None
 
-    @staticmethod
-    def _check_result(result, wanted_list):
+    @classmethod
+    def _check_result(cls, result, wanted_list):
```

Comment:
No need for `cls`. `staticmethod` preferred.

Reply:
it actually shouldn't be part of this class since it's functionality is not relevant. it's simply a forall utility method. I'll replace it by that.